store/tikv/lock_resolver.go

```diff
@@ -566,7 +583,13 @@ func (lr *LockResolver) getTxnStatus(bo *Backoffer, txnID uint64, primary []byte
 			return status, err
 		}
 		status.action = cmdResp.Action
-		if cmdResp.LockTtl != 0 {
+		if cmdResp.LockInfo != nil {
+			status.secondaries = cmdResp.LockInfo.Secondaries
+			status.useAsyncCommit = cmdResp.LockInfo.UseAsyncCommit
+		}
+		if status.useAsyncCommit {
+			status.commitTS = cmdResp.LockInfo.MinCommitTs
```

Comment:
I believe there should be a comment because although we set `commitTS` to `MinCommitTS`, the status of the txn is not determined, the secondary locks need to be checked to determine the final txn status.

Reply:
There is a comment on `TxnStatus::commitTS`, I can add one here too